### ISSUE TITLE
Slider and RangeSlider layout is broken in Firefox

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1301]
We have inline style "**height: '-webkit-fill-available'**" for the SliderLabel, which does not work in **Firefox**. 
The issue can be observed in all of our demos for the RangeSlider and the Slider:
[https://www.telerik.com/kendo-react-ui/components/inputs/slider/](https://www.telerik.com/kendo-react-ui/components/inputs/slider/)

Adding explicit height seems to resolve the issue (at least as a temporary workaround):
```
            <style>
            .k-slider-track-wrap .k-tick.k-tick-large{
                height: 25px;
            } 
```

[https://stackblitz.com/edit/react-qdweg2-a2nsxc?file=package.json,index.html](https://stackblitz.com/edit/react-qdweg2-a2nsxc?file=package.json,index.html)

**Comment by Xizario** [comment]
The li elements should have `k-slider-items` class and inline `position: absolute` and `height` should be removed.

**Comment by nstoychev** [comment]
Fixed in [v5.4.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#cca1aad7-f941-57a1-be41-be469aecbaf0).
